jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
There is an existing config file in the `jest` folder I think we can just add the `moduleExtensions` to. 

https://github.com/Path-Check/covid-safe-paths/blob/develop/jest/config.js